cardinal/cardinal.go

```diff
@@ -31,8 +31,37 @@ var (
 // This section aggregates function from other packages such that they are easily accessible
 // via cardinal.<function_name>
 
+// NewSearch is used to create a search object.
+//
+// Usage:
+//
+// cardinal.NewSearch(worldCtx).Exact(search.Component[AlphaTest]())
 var NewSearch = search.NewSearch
 
+// NewLegacySearch allows users to create a Search object with a filter already provided
+// as a property.
+//
+// Usage:
+//
+// cardinal.NewSearchWithFilter(wCtx, filter.Exact(Alpha{}, Beta{}))
+var NewLegacySearch = search.NewLegacySearch
+
+type Search = search.Search
+
+// FilterFunction wrap your component filter function of func(comp T) bool inside FilterFunction to use
+// in search.
+//
+// Usage:
+//
+//	cardinal.NewSearch(worldCtx).
```

Comment:
This documentation doesn't seem to hold true anymore since you are moving the wCtx to the `Each`/`First` instead of NewSearch

Reply:
Yeah I'll update that.